### ISSUE TITLE
[Factory] Replace factory buttons

### ISSUE THREAD

**Issue by slemeur** [eclipse-che/che#4501]
The factory buttons have not been used a lot.
It’s probably good time to remove the service that we have, which was allowing to use logo in the button - and just provide an SVG badge as on dashboard. 
We would remove their mention from docs too.

We also need to replace the factory button that we are providing from the dashboard.

**Comment by skabashnyuk** [comment]
@slemeur is that ok for this issue if we marked
```
    @GET
    @Path("/{id}/image")
```
and 
```
    @POST
    @Consumes(MULTIPART_FORM_DATA)
```
as deprecated 
Is that the goal of this issue?

**Comment by slemeur** [comment]
Yes for deprecating.

I will also update the buttons styles (I planned to do that today)

**Comment by slemeur** [comment]
Close: will be fixed with #2049
